### PR TITLE
[zh] sync concepts\services-networking\topology-aware-routing.md

### PR DESCRIPTION
**Pull request by KKtheGhost**
## PR Summary

- **EN upstream**: `content/en/docs/concepts/services-networking/topology-aware-routing.md`
- **Sync to**: `content/zh-cn/docs/concepts/services-networking/topology-aware-routing.md`

Sync check by `scripts/lsync.sh` on `main` branch:
```diff
$ scripts/lsync.sh content/zh-cn/docs/concepts/services-networking/topology-aware-routing.md
diff --git a/content/en/docs/concepts/services-networking/topology-aware-routing.md b/content/en/docs/concepts/services-networking/topology-aware-routing.md
index 1977f199d0..7092946e85 100644
--- a/content/en/docs/concepts/services-networking/topology-aware-routing.md
+++ b/content/en/docs/concepts/services-networking/topology-aware-routing.md
@@ -154,10 +154,9 @@ zone.

 ## Constraints

-* Topology Aware Hints are not used when either `externalTrafficPolicy` or
-  `internalTrafficPolicy` is set to `Local` on a Service. It is possible to use
-  both features in the same cluster on different Services, just not on the same
-  Service.
+* Topology Aware Hints are not used when `internalTrafficPolicy` is set to `Local`
+  on a Service. It is possible to use both features in the same cluster on different
+  Services, just not on the same Service.

 * This approach will not work well for Services that have a large proportion of
   traffic originating from a subset of zones. Instead this assumes that incoming
```
Sync check by`scripts/lsync.sh` on `sync/topology-aware-routing` branch:
```diff
$ scripts/lsync.sh content/zh-cn/docs/concepts/services-networking/topology-aware-routing.md
content/zh-cn/docs/concepts/services-networking/topology-aware-routing.md is still in sync
```

Thanks to the reviewers in advance.
Best Regards,

Kivinsae Fang

### CODE REVIEW

File: content/zh-cn/docs/concepts/services-networking/topology-aware-routing.md [hunk 1]
```diff
@@ -268,13 +268,13 @@ Kubernetes 控制平面和每个节点上的 kube-proxy 在使用拓扑感知提
...
-  `internalTrafficPolicy` is set to `Local` on a Service. It is possible to use
-  both features in the same cluster on different Services, just not on the same
-  Service.
+* Topology Aware Hints are not used when `internalTrafficPolicy` is set to `Local`
+  on a Service. It is possible to use both features in the same cluster on different
+  Services, just not on the same Service.
 -->
-* 当 Service 的 `externalTrafficPolicy` 或 `internalTrafficPolicy` 设置值为 `Local` 时，
-  系统将不使用拓扑感知提示信息。你可以在同一集群中的不同服务上使用这两个特性，但不能在同一个服务上这么做。
+* 当 Service 的 `internalTrafficPolicy` 设置值为 `Local` 时，
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/topology-aware-routing.md:275]:
```suggestion
* 当 Service 的 `internalTrafficPolicy` 值设置为 `Local` 时，
```

File: content/zh-cn/docs/concepts/services-networking/topology-aware-routing.md [hunk 2]
```diff
@@ -268,13 +268,13 @@ Kubernetes 控制平面和每个节点上的 kube-proxy 在使用拓扑感知提
...
+  系统将不使用拓扑感知提示信息。你可以在同一集群中的不同服务上使用这两个特性，
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/topology-aware-routing.md:276]:
```suggestion
  系统将不使用拓扑感知提示信息。你可以在同一集群中的不同 Service 上使用这两个特性，
```

File: content/zh-cn/docs/concepts/services-networking/topology-aware-routing.md [hunk 3]
```diff
@@ -268,13 +268,13 @@ Kubernetes 控制平面和每个节点上的 kube-proxy 在使用拓扑感知提
...
+  但不能在同一个服务上这么做。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/topology-aware-routing.md:277]:
```suggestion
  但不能在同一个 Service 上这么做。
```

**Review by windsonsea — COMMENTED** [review]
一句话内，如果第一个 Service 不翻，那另外 2 个也不用翻。

**Comment by KKtheGhost** [comment]
@windsonsea 
Thanks so much for the code review.
All the updates are committed and rebased.

Best Regards,

**Pull request merged**